examples/dreamcast/vmu/vmu_lcd/lcd.c

```diff
@@ -0,0 +1,186 @@
+/*  KallistiOS ##version##
+
+    lcd.c
+    Copyright (C) 2023 Paul Cercueil
+
+*/
+
+/*
+    This example demonstrates drawing dynamic contents to the
+    VMU's LCD display. It does so by rendering to a virtual
+    framebuffer and then presenting it, which sends the updated
+    framebuffer to the VMU over the maple protocol which displays
+    it. 
+
+    This demo also shows off rendering dynamic text using an 
+    embedded font.
+ */
+
+#include <dc/maple.h>
+#include <dc/maple/controller.h>
+#include <dc/maple/vmu.h>
+#include <dc/vmu_fb.h>
+
+#include <stdio.h>
+#include <math.h>
+#include <stdint.h>
+
+/* 4x6 font from the Linux kernel:
+ * https://github.com/torvalds/linux/blob/master/lib/fonts/font_mini_4x6.c
+ *
+ * Modified locally to pack the data better. */
```

Comment:
This is a bit problematic, since it is using Linux kernel sources, and thus is GPLed.

Either the example needs to be marked such that everyone knows that section of code is GPLed (and thus the example itself is covered by the GPL and anything else using that font is), or we need to replace the font.

Reply:
I thought this one was actually okay, just because the source file has this at the top:

```
/* Hand composed "Minuscule" 4x6 font, with binary data generated using
 * Perl stub.
 *
 * Use 'perl -x mini_4x6.c < mini_4x6.c > new_version.c' to regenerate
 * binary data.
 *
 * Created by Kenneth Albanowski.
 * No rights reserved, released to the public domain.
 *
 * Version 1.0
 */
```

If we're still at GPL risk, though, definitely agree, we can find another font.

Reply:
Yep, the font is public domain.
We can add that info to the header.

Reply:
Could we add the original creator's name to the comment too, please. In fact, just copy the Created by and public domain lines from the comment into the example -- out of respect to the original creator. Please keep the part about it being modified as well, of course.

Reply:
Sounds good! Done.